src/fs/rename.js

```diff
@@ -1,5 +1,24 @@
+import { throwError } from "./error.js";
+import { access, rename as renameFile } from 'fs/promises';
+
 const rename = async () => {
-    // Write your code here 
+    try {
+        const WRONG_FILE_NAME = 'wrongFilename.txt',
+            PROPER_FILE_NAME = 'properFilename.md',
+            PATH = './files'
+
+        
+        await access(`${PATH}/${PROPER_FILE_NAME}`)
+            .then(() => {
+                console.log('file exist')
+                throwError()
```

Comment:
this doesn't work as expected, exception isn't thrown (because it's thrown and then caught)

Reply:
it is check file access and throw error only when it already access. It is in a task description.
rename.js - implement function that renames file wrongFilename.txt to properFilename with extension .md (if there's no file wrongFilename.txt or properFilename.md already exists Error with message FS operation failed must be thrown)

Reply:
I meant you can delete this code

>         await access(`${PATH}/${PROPER_FILE_NAME}`)
>             .then(() => {
>                 console.log('file exist')
>                 throwError()
>             }).catch(() => {})

This code throws an exception and then catches with empty callback, so it useless